worker.sh

```diff
@@ -1,3 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-echo "hello world"
\ No newline at end of file
+wget ${PBF_URL} -O data.pbf
+python osm-fudge/amenity-example.py data.pbf
```

Comment:
Hm I think we should call it with `python3`; and also install Python 3 in the Dockerfile.

By default I think it's still Python 2.7 (?)

Reply:
Oh yes. The application part of it runs on python3 whereas preparing data part is python2 based. Going to make it consistent to python3